worlds/smw/Client.py

```diff
@@ -405,6 +529,15 @@ async def game_watcher(self, ctx):
                     snes_buffered_write(ctx, SMW_SFX_ADDR, bytes([item_rom_data[item.item][2]]))
 
                 snes_buffered_write(ctx, WRAM_START + item_rom_data[item.item][0], bytes([new_item_count]))
+            elif item.item in icon_rom_data:
+                queue_addr = await snes_read(ctx, WRAM_START + icon_rom_data[item.item][0], 2)
+                queue_addr = queue_addr[0]+(queue_addr[1]<<8)
```

Comment:
I find these instances where there are no spaces around operators to be cramped, and I'd bet it's not PEP8, but it's not worth blocking the PR over.